src/main/java/edu/kis/vh/nursery/collection/IntArrayStack.java

```diff
@@ -9,7 +9,7 @@
     @Override
     public void push(final int in) {
         if (!isFull())
-            numbers[++total] = in;
+            numbers[total++] = in;
```

Comment:
Changing DEFAULT_EMPTY_STACK_VALUE value requires such fixes? It does not make sense. Incorrect fix.